dictionaries/en_GB/src/additional_words.txt

```diff
@@ -15,5 +15,9 @@ iPads
 prepend
 reauthenticate
 tricep
+uncomment
```

Comment:
This exists in the the software terms https://github.com/streetsidesoftware/cspell-dicts/blob/30f6a0cc588f211b0ec7c43e956d3edeff263cbd/dictionaries/software-terms/src/software-terms.txt#L1386 but not the variations

Reply:
Yeah, I agree, it probably belongs to software-terms, good catch, I updated PR